src/txmempool.cpp

```diff
@@ -22,6 +23,22 @@
 #include <cmath>
 #include <optional>
 
+template <typename T>
+static void MaybeAdd(T& mut, const T& apply)
+{
+    if (AdditionOverflow(mut, apply)) {
+        // Signed integer overflow is UB in theory.
+        // It is impossible to predict when and if an overflow occurs, since
+        // the overflow caused by a prioritisetransaction RPC might only be
+        // later hit when descendant txs are added to the mempool.
+        // Since it is impossible to predict reliably, leave it up to the user
+        // to use the RPC endpoint responsibly, considering their mempool
+        // limits and usage patterns.
+        return;
```

Comment:
This ignores the fee delta when it would otherwise overflow, which seems like incorrect behaviour. Shouldn't we just set `mut` to the max value?

Reply:
thanks, done